components/brave_rewards/browser/rewards_service.h

```diff
@@ -57,6 +57,7 @@ using GetPublisherAllowVideosCallback = base::Callback<void(bool)>;
 using GetAutoContributeCallback = base::Callback<void(bool)>;
 using GetReconcileStampCallback = base::Callback<void(uint64_t)>;
 using IsWalletCreatedCallback = base::Callback<void(bool)>;
+using GetPendingContributionsTotalCallback = base::Callback<void(double)>;
```

Comment:
this should be `base::CallbackOnce` and update to use `base::BindOnce`. `base::Callback` and `base::Bind` are deprecated in favor of Once/Repeating

Reply:
fyi - you need to use std::move when passing or calling a `base::CallbackOnce` callback

Reply:
I tried to add it, but it's failing for me, will contact you via DM